.dockerignore

```diff
@@ -1,30 +1,6 @@
-abc
-ace2
-build
-cmake
-CMakeLists.txt
-deploy_key.enc
-docker
-Dockerfile
-docs
-libopenfpga
-libs
-LICENSE
-Makefile
-openfpga
-README.md
-run_local.bat
-run_local.sh
-vpr
-yosys
+# Ignore everything
+**
```

Comment:
It looks like your docker will ignore everything? If this is a pull request about documentation, I do not think it should include anything beyond ``doc`` directory.

Reply:
yeah while building we want to ignore anything other than installation scripts in `.github/*`